testsuite/features/secondary/https_connection.feature

```diff
@@ -11,12 +11,12 @@ Feature: Secure connection
   Scenario: Connecting to the server securely
     When I connect to the server securely
     Then the connection should be secured
-    And the page title should be "SUSE Multi-Linux Manager - Sign In" text
+    Then the page title should be "SUSE Multi-Linux Manager - Sign In" text
 
   Scenario: Connecting to the server securely while using CA certificate file
     When I connect to the server securely while using CA certificate file
     Then the connection should be secured
-    And the page title should be "SUSE Multi-Linux Manager - Sign In" text
+    Then the page title should be "SUSE Multi-Linux Manager - Sign In" text
```

Comment:
Here we want `And`, as we had a previous `Then`.
While the Step definition on Ruby, should say `Then`.